script.js

```diff
@@ -0,0 +1,54 @@
+let numberOne = ""
+let operator = ""
+let numberTwo = ""
+
+
+function addition(n1,n2) {
+    return n1 + n2
+}
+
+function subtraction(s1,s2) {
+    return s1 - s2
+}
+
+function division(d1,d2) {
+    return d1 / d2
+}
+
+function multiplication(m1,m2) {
+    return m1 * m2
+}
```

Comment:
We usually create methods in the infinitive. So, `addition` will be better named as `add` etc...